app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -52,11 +54,20 @@ public Account(String location, String name, String phoneNumber) {
         this.name = name;
         this.phoneNumber = phoneNumber;
         this.cart = new Cart();
+        currentAccount = this;
 //        this.history = new History();
     }
 
     /* ********* Getters & Setters: ********* */
 
+    public static Account getCurrentAccount() {
```

Comment:
Maybe we can save only the uid instead of all the object and then you can write it in the db.

Pay attention that every change in any object needs to be written in the db

Reply:
we did it static because of efficiency considerations - it leads to less queries from the db